### PR TITLE
Enhance password checks (Closes: #615, #625)

### PR DESCRIPTION
**Pull request by Moppa5**
#### What are the main changes in this PR?

- Format/Filter the wp-check-passwords output
- Remove return code from the wp-check-haveibeenpwned

##### Why are we doing this? Any context or related work?
Mostly related to #615, but because both are related to password checking also #625 minor fix included.

### CODE REVIEW

File: src/modules/pages/security.php [hunk 1]
```diff
@@ -142,6 +151,24 @@ public static function init_postboxes( Toolpage $page ) {
...
+   */
+  public static function check_passwords() {
+    $check_passwords = Compatibility::exec('wp-check-passwords', $output, $ret_val);
+
+    if ( $check_passwords === false ) {
+      return Ajax\AjaxResponse::command_error_response('wp-check-passwords', $ret_val);
+    }
+    // Filter and format the color chars from output.
+    $pattern = '/\x1b\[[0-9;]*m/';
+    $output = \preg_replace($pattern, '', $output);
```
Comment (JoosuaKoskinen) [inline src/modules/pages/security.php:166]: `preg_replace` returns `null` on error

File: src/modules/pages/security.php [hunk 2]
```diff
@@ -5,9 +5,10 @@
 use \Seravo\Helpers;
 use \Seravo\CruftRemover;
 
+use \Seravo\Ajax;
 use \Seravo\Ajax\AjaxHandler;
 use \Seravo\Ajax\AjaxResponse;
-
+use Seravo\Compatibility;
```
Comment (JoosuaKoskinen) [inline src/modules/pages/security.php:11]: This should be prefixed with `\` like the others

**Review by JoosuaKoskinen — CHANGES_REQUESTED** [review]
Feel free to merge after these changes

File: src/modules/pages/security.php (hunk 2 above)
Comment (Moppa5) [reply src/modules/pages/security.php:11]: Yup. Most likely VSC added this automatically and I forgot to re-check that.

File: src/modules/pages/security.php (hunk 1 above)
Comment (Moppa5) [reply src/modules/pages/security.php:166]: Good point!

**Pull request merged**